### PR TITLE
Document end-to-end benchmarking with Caffe2

### PR DESCRIPTION
**Pull request by Maratyszcza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,68 @@ To cross-compile for iOS, clone [ios-cmake](https://github.com/leetal/ios-cmake)
...
+adb push predict_net.pb /data/local/tmp/predict_net.pb
+
+# Run speed benchmark with 50 warm-up iterations and 10 measurement iterations
+adb shell /data/local/tmp/speed_benchmark \
+	--net /data/local/tmp/predict_net.pb \
+	--init_net /data/local/tmp/init_net.pb \
+	--input data --input_dims 1,3,224,224 --input_type float \
+	--warmup 50 --iter 10
+```
+
```
Comment (harouwu) [inline README.md]: Can we add another chapter saying about PEP methods? I can do that

File: README.md [hunk 2]
```diff
@@ -37,6 +37,68 @@ To cross-compile for iOS, clone [ios-cmake](https://github.com/leetal/ios-cmake)
...
+```bash
+# Clone PyTorch 1.0 repo
+git clone --recursive https://github.com/pytorch/pytorch.git
+cd pytorch
+
+# Optional: update QNNPACK submodule to latest revision
+git submodule update --remote third_party/QNNPACK
+
+# Build Caffe2 (including binaries) for Android
+scripts/build_android.sh -DANDROID_TOOLCHAIN=clang -DBUILD_BINARY=ON
```
Comment (harouwu) [inline README.md]: btw, I have been using confu + ninja, what is our plan on it?

File: README.md [hunk 3]
```diff
@@ -37,6 +37,68 @@ To cross-compile for iOS, clone [ios-cmake](https://github.com/leetal/ios-cmake)
 | i386         | `scripts/build-ios-i386.sh`   | iPhone Simulator (32-bit) |
 | x86_64       | `scripts/build-ios-x86_64.sh` | iPhone Simulator (64-bit) |
 
+## End-to-End Benchmarking
+
+Caffe2 backend of PyTorch 1.0 natively integrates QNNPACK, and provides a [pre-trained quantized MobileNet v2 model](https://github.com/caffe2/models/tree/master/mobilenet_v2_quantized). Below are instructions for benchmarking this model end-to-end with QNNPACK.
```
Comment (harouwu) [inline README.md]: Shall we talk about this is about to reproduce the blog post results?

File: README.md (hunk 1 above)
Comment (Maratyszcza) [reply README.md]: please do, I'm not familiar with PEP workflow

File: README.md (hunk 2 above)
Comment (Maratyszcza) [reply README.md]: this builds Caffe2, not QNNPACK. QNNPACK has build scripts for both confu+ninja and CMake.

File: README.md (hunk 1 above)
Comment (harouwu) [reply README.md]: I will do after you merge this. Let's merge it fast

File: README.md (hunk 3 above)
Comment (Maratyszcza) [reply README.md]: yes, if you can add details on TFLite benchmarking, please do

**Pull request merged**